controllers/auth.controller.js

```diff
@@ -53,13 +56,28 @@ const signup = async (req, res, next) => {
   }
   try {
     const newUser = new User({ email });
+    const verificationToken = uuid();
+
     newUser.setPassword(password);
-    await newUser.save();
+    newUser.avatarURL = gravatar.url(email).slice(2);
+    newUser.verificationToken = verificationToken;
+    const result = await newUser.save();
+    const verificationLink = `${req.protocol}://${req.get(
```

Comment:
Można też w pliku .env umieścić jakąś ścieżkę która na przykład będzie tam zapisana nie tworzona dynamicznie to jako stała że tam będzie weryfikacja.